sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -272,13 +272,23 @@ public Builder Server(String domain) {
          * @return a new Octane instance which has the set context and is correctly authenticated
          */
         public Octane build() {
+            return this.build(false);
+        }
+
+        /**
+         * The main build procedure which creates the {@link Octane} object and authenticates against the server
+         *
+         * @param doNotValidateCertificate - Disables validating server SSL certificates
+         * @return a new Octane instance which has the set context and is correctly authenticated
+         */
+        public Octane build(final boolean doNotValidateCertificate) {
```

Comment:
IMO this flag should be a step in the build process, with a default value. Going by our current "convention" I guess it would be a method called SkipSslCertificateValidation(boolean skipCertificateValidation).

Reply:
I need to think about this one